frontend/packages/metal3-plugin/locales/zh/metal3-plugin.json

```diff
@@ -0,0 +1,202 @@
+  "Management Address": "管理地址",
+  "Add credentials": "添加凭证",
+  "View all": "查看所有",
+  "Host name": "主机名",
+  "View events": "查看事件",
+  "There are no ongoing activities.": "没有正在进行的活动。",
+  "Inventory": "清单",
+  "Disk": "磁盘",
+  "NIC": "NIC",
+  "Memory": "内存",
+  "Disk Usage": "磁盘用量",
+  "Utilization": "使用率",
+  "Filesystem": "文件系统",
+  "Network Transfer": "网络传输",
+  "Pod count": "Pod 数量",
+  "Start Maintenance": "开始维护",
+  "Stop Maintenance": "停止维护",
+  "Power On": "开机",
+  "Deprovision": "取消置备",
+  "Deprovision {{name}}": "取消置备 {{name}}",
+  "Are you sure you want to delete {{name}} machine and scale down its machine set?": "您确定要删除 {{name}} 机器并缩减其机器集吗？",
+  "Are you sure you want to delete {{name}} machine?": "您确定要删除 {{name}} 机器吗？",
+  "Power Off": "关机",
+  "Restart": "重启",
+  "Delete Bare Metal Host": "删除裸机主机",
+  "Registering": "注册",
+  "Available": "可用",
+  "Provisioning": "置备",
+  "Provisioned": "已置备",
+  "Error": "错误",
+  "Maintenance": "维护",
+  "Unmanaged": "非受管",
+  "Other": "其他",
+  "Node Details": "节点详情",
+  "Node Name": "节点名称",
+  "External ID": "外部 IP",
+  "Node Addresses": "节点地址",
+  "Node Labels": "节点标签",
+  "Taints": "污点",
+  "Annotations": "注解",
+  "Bare Metal Host": "裸机主机",
+  "Provider ID": "供应商 ID",
+  "Created": "创建的",
+  "Operating System": "操作系统",
+  "OS Image": "OS 镜像",
+  "Architecture": "架构",
+  "Kernel Version": "内核版本",
+  "Boot ID": "引导 ID",
+  "Container Runtime": "容器运行时",
+  "Kubelet Version": "Kubelet 版本",
+  "Kube-Proxy Version": "Kube-Proxy 版本",
+  "{{name}} · Details": "{{name}}详情",
+  "{{name}} Details": "{{name}} 详情",
+  "Nodes": "节点",
+  "This node has requested to join the cluster. After approving its certificate signing request the node will begin running workloads.": "此节点请求加入集群。批准其证书签名请求后，节点将开始运行工作负载。",
+  "This node has a pending server certificate signing request. Approve the request to enable all networking functionality on this node.": "此节点有一个待处理的服务器证书签名请求。批准请求，在该节点上启用所有联网功能。",
+  "Certificate approval required": "需要对证书进行批准",
+  "Request": "要求",
+  "Deny": "拒绝",
+  "Image": "镜像",
+  "Images": "镜像",
+  "Bare Metal Operator not available": "裸机 Operator 不可用",
+  "The Bare Metal Operator that enables this capability is not available or disabled.": "启用此功能的逻辑 Operator 不可用或已禁用。",
+  "Power management not available": "电源管理不可用",
+  "Power operations cannot be performed on this host until Baseboard Management Controller (BMC) credentials are provided for the underlying host.": "在为底层主机提供 Baseboard Management Controller(BMC)凭证前，电源操作不能在这个主机上执行。",
+  "Approve Server CSR": "批准服务器 CSR",
+  "Approve Node Server CSR": "批准节点服务器 CSR",
+  "Are you sure you want to approve server CSR for {{name}}?": "您确定要为 {{name}} 批准服务器 CSR 吗?",
+  "Approve": "批准",
+  "Ready": "就绪",
+  "Not Ready": "未就绪",
+  "Approval Required": "需要批准",
+  "Starting maintenance": "开始维护",
+  "Node is entering maintenance. The cluster will automatically rebuild node&apos;s data 30 minutes after entering maintenance.": "节点正在进入维护。集群将在进入维护后 30 分钟自动重建节点的数据。",
```

Comment:
should `&apos;` be `'`

Reply:
I can't update these file keys without touching the metal3-plugin code. I can file a separate bug for these.

Reply:
See https://bugzilla.redhat.com/show_bug.cgi?id=1908434.

Reply:
PR merged; this is all set now.